src/main/java/org/jabref/gui/GUIGlobals.java

```diff
@@ -156,6 +157,16 @@ public static void init() {
         GUIGlobals.currentFont = new Font(Globals.prefs.get(JabRefPreferences.FONT_FAMILY),
                 Globals.prefs.getInt(JabRefPreferences.FONT_STYLE), Globals.prefs.getInt(JabRefPreferences.FONT_SIZE));
 
+        // Set WM_CLASS using reflection for certain Un*x window managers
+        try {
```

Comment:
I guess we should only execute this code, if we are actually running on a unix system, right? That means it should all be wrapped in an `if (OS.LINUX)`

Reply:
This would also exclude Unix systems like FreeBSD, wouldn't it?

Reply:
In all my time in the JabRef team, I've never heard of someone running it on FreeBSD. Anyway, the alternative solution is `!OS.WINDOWS && !OS.OSX` :)

Reply:
I added that alternative variant now. Thanks for your review!